File: app/src/main/java/com/me/bui/architecturecomponents/util/UtilBindings.java
package com.me.bui.architecturecomponents.util;

import android.content.Context;
import android.databinding.BindingAdapter;
import android.view.View;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

/**
 * Created by mao.bui on 9/3/2018.
 */
public class UtilBindings {
    @BindingAdapter("visibleGone")
    public static void showHide(View view, boolean show) {
        view.setVisibility(show ? View.VISIBLE : View.GONE);
    }

    @BindingAdapter("imageUrl")
    public static void bindImage(ImageView imageView, String url) {
        Context  context = imageView.getContext();
        Glide.with(context)
                .load(url)
                .into(imageView);
    }
}

File: app/src/main/java/com/me/bui/architecturecomponents/util/UtilBindings.kt
package com.me.bui.architecturecomponents.util

import android.databinding.BindingAdapter
import android.util.Log
import android.view.View
import android.widget.ImageView
import com.bumptech.glide.Glide

/**
 * Created by mao.bui on 9/3/2018.
 */
object UtilBindings {
    @JvmStatic
    @BindingAdapter("visibleGone")
    fun showHide(view: View, show: Boolean) {
        view.visibility = if (show) View.VISIBLE else View.GONE
    }

    @JvmStatic
    @BindingAdapter("imageUrl")
    fun bindImage(imageView: ImageView?, url: String?) {
        val context = imageView!!.context
        Log.d("URL" , " $url")
        Glide.with(context)
                .load(url)
                .into(imageView)
    }
}
